.git-blame-ignore-revs

```diff
@@ -0,0 +1,5 @@
+# For more information, see:
+# https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
+
+# Black code formatting of entire repository
+c75775b10b19a361fd8ecfdaccc2bb2f820ea115
```

Comment:
Please check this commit hash matches the one for that implements black across the whole repo. I'm super paranoid I've got the wrong one!